### PR TITLE
LPS-168035 buildLang

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://issues.liferay.com/browse/LPS-168034
Related issue: https://issues.liferay.com/browse/LPS-168035
Related issue: https://issues.liferay.com/browse/LPS-168036

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:relevant

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -11001,10 +11001,12 @@ object=Object
 object-action-executor-help[add-object-entry]=Add entry on related or unrelated objects.
 object-action-executor-help[groovy]=Create custom actions using Groovy.
 object-action-executor-help[notification]=Send a notification template.
+object-action-executor-help[update-object-entry]=Update the current entry
```
Comment (carolmariaabb) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:11004]: @marcelabc This pull request LGTM, but maybe Brian will ask If this a full sentence (the end of a complete sentence should be marked by a period..) or a Title (capitalize the beginning of every word (?)) 🤦🏽‍♀️

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Review by carolmariaabb — APPROVED** [review]

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
Resent here: #1359

**Pull request closed without merging**